### PR TITLE
{data}[foss/2023a] dask v2023.12.1

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/d/dask/dask-2023.12.1-foss-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+use_pip = True
+
+exts_list = [
+    ('toolz', '0.12.0', {
+        'checksums': ['88c570861c440ee3f2f6037c4654613228ff40c93a6c25e0eba70d17282c6194'],
+    }),
+    ('locket', '1.0.0', {
+        'checksums': ['5c0d4c052a8bbbf750e056a8e65ccd309086f4f0f18a2eac306a8dfa4112a632'],
+    }),
+    ('partd', '1.4.0', {
```
Comment (boegel) [inline easybuild/easyconfigs/d/dask/dask-2023.12.1-foss-2023a.eb:29]: @pavelToman There's a 1.4.1 release of `partd`, let's update to that?

File: easybuild/easyconfigs/d/dask/dask-2023.12.1-foss-2023a.eb [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        'checksums': ['aa0ff35dbbcc807ae374db56332f4c1b39b46f67bf2975f5151e0b4186aed0d5'],
+    }),
+    ('HeapDict', '1.0.1', {
+        'checksums': ['8495f57b3e03d8e46d5f1b2cc62ca881aca392fd5cc048dc0aa2e1a6d23ecdb6'],
+    }),
+    ('zict', '3.0.0', {
+        'checksums': ['e321e263b6a97aafc0790c3cfb3c04656b7066e6738c37fffcca95d803c9fba5'],
+    }),
+    ('tblib', '2.0.0', {
```
Comment (boegel) [inline easybuild/easyconfigs/d/dask/dask-2023.12.1-foss-2023a.eb:38]: should be updated to `3.0.0`?

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen2

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3124.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/816326e0df6ed572f51e50c77385aeb3 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pavelToman!

**Pull request merged**